crates/nu-command/tests/commands/into_filesize.rs

```diff
@@ -63,8 +63,57 @@ fn into_filesize_negative_filesize() {
 }
 
 #[test]
-fn into_negative_filesize() {
+fn into_filesize_negative_str_filesize() {
+    let actual = nu!("'-3kib' | into filesize");
+
+    assert!(actual.out.contains("-3.0 KiB"));
+}
+
+#[test]
+fn into_filesize_wrong_negative_str_filesize() {
+    let actual = nu!("'--3kib' | into filesize");
+
+    assert!(actual.err.contains("can't convert string to int"));
+}
+
+#[test]
+fn into_filesize_negative_str() {
     let actual = nu!("'-1' | into filesize");
 
     assert!(actual.out.contains("-1 B"));
 }
+
+#[test]
+fn into_filesize_wrong_negative_str() {
+    let actual = nu!("'--1' | into filesize");
+
+    assert!(actual.err.contains("can't convert string to int"));
+}
+
+#[test]
+fn into_filesize_positive_str_filesize() {
+    let actual = nu!("'+1Kib' | into filesize");
+
+    assert!(actual.out.contains("1.0 KiB"));
+}
+
+#[test]
+fn into_filesize_wrong_positive_str_filesize() {
+    let actual = nu!("'++1Kib' | into filesize");
+
+    assert!(actual.err.contains("can't convert string to int"));
+}
```

Comment:
Thanks for all the tests! Unfortunately, the expected output will need to be updated because of the previous comment.

Reply:
Yeah, i also fixed testcases too.